ci/glci/model.py

```diff
@@ -316,7 +323,8 @@ class ReleaseManifest(ReleaseIdentifier):
     build_timestamp: str
     paths: typing.Tuple[typing.Union[S3_ReleaseFile]]
     published_image_metadata: typing.Union[AlicloudPublishedImageSet,
-                                           AwsPublishedImageSet, AzurePublishedImage, GcpPublishedImage, None]
+                                           AwsPublishedImageSet, AzurePublishedImage,
```

Comment:
as discussed yesterday, I recommend formatting list member one entry per line and order alphabetically

Reply:
you don't use an auto-formatter like black [1] ?

[1] https://github.com/psf/black